src/mainwindow.cpp

```diff
@@ -293,7 +293,7 @@ void MainWindow::bringToFront()
         return;
     m_scene->setCursorVisible(false);
 
-    QGraphicsItem *selectedItem = m_scene->selectedItems().first();
+    QGraphicsItem *selectedItem = m_scene->selectedItems().constFirst();

```

Comment:
in line 304, this obkect is used to write to. So constFirst is probably not a good idea to use.